src/test/java/it/pagopa/pn/cucumber/steps/SharedSteps.java

```diff
@@ -197,7 +200,9 @@ public void destinatarioMarioCucumber() {
                         .taxId(marioCucumberTaxID)
                         .digitalDomicile(new NotificationDigitalAddress()
                                 .type(NotificationDigitalAddress.TypeEnum.PEC)
-                                .address("testpagopa3@pnpagopa.postecert.local")));
+
+                                .address("notifichedigitali-uat@pec.pagopa.it")));
```

Comment:
converrebbe che la mail sia parametrica non Hard coded data la necessità di variarla in base all'ambiente